py/desitarget/mock/selection.py

```diff
@@ -20,8 +20,34 @@
 import desispec.brick
 import warnings
 
+
+def make_lookup_dict(bricks):
+    """
+    Creates lookup dictionary for a list of bricks.
+    
+    Parameters:
+    -----------
+    bricks (array): array of bricknames.
+
+    Output:
+    -------
+    Lookup Dictionary. lookup['brickname'] returns a list with all the indices
+    in the input array bricks where bricks=='brickname'
+    """
+    lookup = {}
```

Comment:
FYI, I updated this function to use "lookup" instead of "l" (lowercase ell) as the variable name, since l (ell) can look so much like 1 (one) and I (capital eye).